### DISCUSSION TITLE
Keep review after renovate rebases a PR on GitHub

### DISCUSSION THREAD

**Discussion by roded** [renovatebot/renovate#30316] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi,
Is it possible to have renovate keep reviews after PRs are automatically rebased by renovate?
Thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Not possible - GitHub removes them
